lib/preboot/logger.js

```diff
@@ -12,6 +12,7 @@ module.exports = function (app, options, callback) {
   debug('executed, perform hookable "logger"');
 
   app.hookable('logger', function (ext, done) {
+    app.logProvider = winston;
```

Comment:
``` suggestion
    app.logProvider = {
      transports: winston.transports,
      formats: winston.formats,
      Container: winston.Container
    };
```

If we wanted to be super verbose here we could expose [everything not related to the default logger](https://github.com/winstonjs/winston/blob/master/lib/winston.js#L19-L68) from the `winston` source.